README.md

```diff
@@ -1,31 +1,17 @@
-<a name="readme-top"></a>
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
+ Hello Microverse Project
 
-Modify this file to match your project and remove sections that don't apply.
+> Description of the project.
+This is an HTML & CSS Hello Microverse project
 
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
```

Comment:
- [ ] @EnixWord 👋, Kindly fix the merge conflict on this project to enable your linters run. Merge conflicts happen when people make different changes to the same line of the same file, or when one person edits a file and another person deletes the same file. You must resolve all merge conflicts before you can merge a pull request on GitHub.